gufe/components/proteincomponent.py

```diff
@@ -204,15 +217,21 @@ def _from_openmmPDBFile(cls, openmm_PDBFile: Union[PDBFile, PDBxFile],
             atom_name = a.GetMonomerInfo().GetName()
 
             connectivity = sum(
-                int(bond.GetBondType()) for bond in a.GetBonds()
+                _BONDORDER_TO_ORDER[bond.GetBondType()]
+                for bond in a.GetBonds()
             )
             default_valence = periodicTable.GetDefaultValence(atomic_num)
 
             if connectivity == 0:  # ions:
-                if atom_name in positive_ions:
+                if atom_name.upper() in positive_ions:
                     fc = default_valence  # e.g. Sodium ions
-                elif atom_name in negative_ions:
+                elif atom_name.upper() in negative_ions:
                     fc = - default_valence  # e.g. Chlorine ions
+                elif atom_name.strip(string.digits).upper() in positive_ions:
+                    # catches cases like 'CL1' as name
+                    fc = default_valence
+                elif atom_name.strip(string.digits).upper() in negative_ions:
+                    fc = - default_valence
```

Comment:
this could be simplified, right?

```
                if atom_name.strip(string.digits).upper() in positive_ions:
                    # catches cases like 'CL1' as name
                    fc = default_valence
                elif atom_name.strip(string.digits).upper() in negative_ions:
                    fc = - default_valence
```

Reply:
hmm yeah you're right here